smithy-codegen-core/src/test/resources/software/amazon/smithy/codegen/core/definitions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "types": {
+    "t1": "t1val",
+    "t2": "t2val"
+  },
+  "tags": {
+    "tag1": "tag1val",
+    "tag2": "tag2val"
+  }
+}
```

Comment:
Looks like there are still some files hanging around without empty lines at the end.